bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckablePullRequest.java

```diff
@@ -79,13 +79,16 @@ private String commitMessage(Hash head, List<Review> activeReviews, Namespace na
         var reviewers = reviewerNames(eligibleReviews, namespace);
         var currentUser = pr.repository().forge().currentUser();
 
-        if (manualReviewers) {
-            var allReviewers = reviewerNames(activeReviews, namespace);
-            var additionalReviewers = Reviewers.reviewers(currentUser, comments);
-            for (var additionalReviewer : additionalReviewers) {
-                if (!allReviewers.contains(additionalReviewer)) {
-                    reviewers.add(additionalReviewer);
+        if (manualReviewersAndStaleReviewers) {
+            reviewers.addAll(Reviewers.reviewers(currentUser, comments));
```

Comment:
In the old code, if someone did a non approved verdict review and was added manually as a reviewer, they would not get reviewer credit. With this change, we unconditionally add all manual reviewers. That is probably how most people would expect the command to work though. @kevinrushforth do you have an opinion here?

Reply:
Yes, it seems expected that all manually credited reviewers would be added regardless of whether or not they had also done a non-approving review. I presume there will be no change in behavior in the (vast majority) of cases where the PR author has not used the `/reviewer credit` command: only approving reviewers should be added and not those who had done a non-approving review?

Reply:
Correct, that case is unchanged. Since you agree with the new behavior, I'm also ok with it. I did find the old one a bit odd.

Reply:
I was thinking the old one is a bug... So I changed it